common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -366,18 +316,15 @@
       .control {
         @extend %video-control;
 
-        .icon-fallback-img {
-
-          .icon {
-            // if the icon font doesn't render, we need to provide dimensions for the svg's
-            width: 1em;
-            height: 1em;
+        .icon {
```

Comment:
This can be removed too as it only applied to the SVG, which are no longer using.